src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -0,0 +1,945 @@
+#include "ScriptedCreature.h"
+#include "ScriptedGossip.h"
+#include "SharedDefines.h"
+#include "SpellAuras.h"
+#include "SpellAuraEffects.h"
+#include "SpellScript.h"
+#include "SpellMgr.h"
+#include "TemporarySummon.h"
+#include "Unit.h"
+#include "Vehicle.h"
+
+Position const OrgrimmarPortalPos[3] =
+{
+    { 1358.62f, 1054.72f, 53.1200f, 0.0f },
+    { 1393.27f, 1021.20f, 53.2225f, 0.0f },
+    { 1404.71f, 1063.73f, 60.5617f, 0.0f }
+};
+
+Position const HellscreamElitePos[16] =
+{
+    { 1387.90f, 1029.71f, 53.21853f, 2.827433f },
+    { 1389.79f, 1024.51f, 53.20833f, 2.827433f },
+    { 1388.05f, 1026.91f, 53.20833f, 2.827433f },
+    { 1388.16f, 1020.88f, 53.25523f, 2.827433f },
+    { 1385.79f, 1025.99f, 53.22593f, 2.827433f },
+    { 1386.69f, 1023.26f, 53.24393f, 2.827433f },
+    { 1384.33f, 1022.04f, 53.28123f, 2.827433f },
+    { 1391.10f, 1027.73f, 53.20483f, 2.827433f },
+
+    { 1359.10f, 1046.55f, 52.97053f, 5.253441f },
+    { 1360.89f, 1051.81f, 53.19793f, 5.253441f },
+    { 1360.75f, 1048.84f, 53.12893f, 5.253441f },
+    { 1364.43f, 1053.62f, 53.29343f, 5.253441f },
+    { 1363.08f, 1048.15f, 53.22223f, 5.253441f },
+    { 1364.08f, 1050.84f, 53.29163f, 5.253441f },
+    { 1366.69f, 1050.31f, 53.34203f, 5.253441f },
+    { 1357.85f, 1050.12f, 52.99823f, 5.253441f }
+};
+
+Position const GarroshPos = { 1402.45f, 1061.62f, 60.56173f, 3.926991f };
+
+Position const GarroshJumpPos = { 1378.65f, 1044.23f, 53.8389f, 5.51524f };
+
+Position const CromushPos = { 1404.71f, 1063.73f, 60.5617f, 2.827433f };
+
+Position const AgathaPreRisePos = { 1364.02f, 1028.54f, 66.99143f };
+
+Position const AgathaRisePos = { 1368.65f, 1032.19f, 63.3033f };
+
+Position const AgathaPreResetPos = { 1364.02f, 1028.54f, 55.9914f };
+
+Position const AgathaResetPos = { 1364.02f, 1028.54f, 58.1319f };
+
+enum QuestTheWarchiefCometh
+{
+    QUEST_THE_WARCHIEF_COMETH               = 26965,
+
+    NPC_LADY_SYLVANAS_WINDRUNNER_COMETH     = 44365,
+    NPC_AGATHA_COMETH                       = 44608,
+    NPC_GRAND_EXECUTOR_MORTUUS              = 44615,
+    NPC_MALE_FALLEN_HUMAN                   = 44592,
+    NPC_FEMALE_FALLEN_HUMAN                 = 44593,
+    NPC_PORTAL_FROM_ORGRIMMAR               = 44630,
+    NPC_GARROSH_HELLSCREAM                  = 44629,
+    NPC_HIGH_WARLORD_CROMUSH_COMETH         = 44640,
+    NPC_HELLSCREAM_ELITE_COMETH             = 44636,
+    NPC_QUEST_MONSTER_CREDIT                = 44629,
+    NPC_FORSAKEN_WARHORSE_UNPHASED          = 73595,
+
+    SPELL_RAISE_FORSAKEN_COMETH             = 83173,
+    SPELL_AIR_REVENANT_ENTRANCE             = 55761,
+    SPELL_SIMPLE_TELEPORT                   = 12980,
+    SPELL_WELCOME_TO_SILVERPINE_CREDIT      = 83384,
+
+    EVENT_START_SCENE_COMETH                = 1,
+    EVENT_SUMMON_PORTAL_COMETH              = 2,
+    EVENT_SUMMON_GARROSH_COMETH             = 3,
+    EVENT_AGATHA_RAISE_FORSAKEN             = 4,
+    EVENT_SCENE_TALK_COMETH                 = 9,
```

Comment:
define how much space the events take in a comment like `// 9-x`

Reply:
done!